__tests__/app_test_.js

```diff
@@ -171,4 +173,114 @@ describe("GET Error Handling", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then(({ body: { reviews } }) => {
+        expect(reviews).toBeSorted("created_at", { descending: true });
+        reviews.forEach((review) => {
+          expect(review).toEqual(
+            expect.objectContaining({
+              review_id: expect.any(Number),
+              title: expect.any(String),
+              designer: expect.any(String),
+              owner: expect.any(String),
+              review_img_url: expect.any(String),
+              category: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              comment_count: expect.any(Number),
+            })
+          );
+        });
+      });
+  });
+  test("200: Responds with reviews object sorted by the date created in descending order", () => {
+    return request(app)
+      .get("/api/reviews?sort_by=created_at&order=desc")
+      .expect(200)
+      .then(({ body: { reviews } }) => {
+        expect(reviews).toBeSorted("created_at", { descending: true });
+      });
+  });
+
+});
+describe("GET Error Handling", () => {
+  test("400: Responds with Invalid request:  Enter a valid sort by or order error message for an invalid sort by query", () => {
+    return request(app)
+      .get("/api/reviews?sort_by=incorrectquery")
+      .expect(400)
+      .then(({ body: { msg } }) => {
+        expect(msg).toEqual(
+          "Invalid request: Enter a valid sort_by query"
+        );
+      });
+  });
+  test("400: Responds with 'Invalid request:  Enter a valid sort by or order' error message for an invalid order query", () => {
+    return request(app)
+      .get("/api/reviews?sort_by=created_at&order=incorrectrder")
+      .expect(400)
+      .then(({ body: { msg } }) => {
+        expect(msg).toEqual(
+          "Invalid request: Enter a valid order query"
+        );
+      });
+  });
+});
+
+describe("GET /api/reviews/:review_id/comments", () => {
+  test("200: Responds with an array of comment objects for the input review id containing the correct properties", () => {
+    return request(app)
+    .get("/api/reviews/3/comments")
+    .expect(200)
+    .then(({body}) => {
+      expect(body.comments).toBeInstanceOf(Array);
+      expect(body.comments).toHaveLength(3)
+      body.comments.forEach((comment) => {
+        expect(comment).toMatchObject({
+          comment_id:expect.any(Number),
+          votes: expect.any(Number),
+          review_id: expect.any(Number),
+          created_at: expect.any(String),
+          author: expect.any(String),
+          body:expect.any(String)
+        });
+      });
+    });
+  });
+  test("200: Responds with an empty array when an review id exists but has no comments", () => {
+      return request(app)
+      .get("/api/reviews/1/comments")
+      .expect(200)
+      .then(({body}) => {
+        expect(body.comments).toBeInstanceOf(Array);
+        expect(body.comments).toHaveLength(1)
```

Comment:
```suggestion
        expect(body.comments).toHaveLength(0)
```
According to your test description this should be 0. The description is correct and this should have length 0. You'll need to update your endpoint accordingly